pure_changelog.md

```diff
@@ -11,6 +11,7 @@
 * Added serial number and timestamp to crashdump filename
 * Changed order of starting services, ServiceDesktop moved to the back
 * Shortened duration of phone modes pop-up from 3s to 1s
+* Added SMS template call rejection window for case when no templates are defined
```

Comment:
shouldn't one of them be removed?

Reply:
You mean the doubled changelog items? :) I thought about it and came to a conclusion that it's hard to contain that changes in a single item: this is a fix on one hand, but adding a new thing on the other hand. But, if more people would like to combine these items into one, I'll do it

Reply:
alright, let's keep it then in two separate entries